onnxruntime/core/providers/xnnpack/xnnpack_execution_provider.cc

```diff
@@ -47,7 +48,7 @@
 class ONNX_OPERATOR_TYPED_KERNEL_CLASS_NAME(kXnnpackExecutionProvider, kMSInternalNHWCDomain, 10, int8_t, QLinearConv);
 class ONNX_OPERATOR_KERNEL_CLASS_NAME(kXnnpackExecutionProvider, kMSInternalNHWCDomain, 1, QLinearAveragePool);
 class ONNX_OPERATOR_KERNEL_CLASS_NAME(kXnnpackExecutionProvider,
-                                      kMSDomain, 1, QLinearSoftmax);
+                                      kMSInternalNHWCDomain, 1, QLinearSoftmax);
 
```

Comment:
The internal NHWC domain should only be for layout sensitive ops. If we're generating our own stub schema on the fly, we should use a domain name defined locally here. AFAIK the domain is an arbitrary string, so even using 'xnnpack' for the domain should be fine.

Reply:
I will create a dynamic domain by 
`constexpr const char* kDynamicDomainByCreate = __DATE__ ".domain";
`